index.html

```diff
@@ -14,6 +14,10 @@
         <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
         <link href="assets/vendor/remixicon/remixicon.css" rel="stylesheet">
         <link href="assets/css/style.css" rel="stylesheet">
+
+        <link href="assets/img/favicons/favicon.ico" rel="icon"/>
+        <link href="assets/img/favicons/favicon-16x16.png" rel="icon" sizes="32x32" type="image/png">
+        <link href="assets/img/favicons/favicon-32x32.png" rel="icon" sizes="16x16" type="image/png">
```

Comment:
Where are these favicons that you added? Did you forget to commit & push them?

<img width="244" alt="Screenshot 2021-12-31 at 00 16 49" src="https://user-images.githubusercontent.com/12160621/147791416-8db2ebac-32ec-4ddd-9b4e-b32174bd34ba.png">

Reply:
Done